test/framework/ressources/prometheus-cluster-role-binding.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why not use `go/src/github.com/coreos/prometheus-operator/example/rbac/prometheus/prometheus-cluster-role-binding.yaml`?